pkg/cmd/repo/autolink/autolink.go

```diff
@@ -0,0 +1,19 @@
+package autolink
+
+import (
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func NewCmdAutolink(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "autolink <command>",
+		Short: "Manage autolink references",
+		Long:  "Work with GitHub autolink references.",
```

Comment:
I think it would be helpful to add some links to relevant documentation. I've added the `heredoc` import and updated the Long doc field to show what I mean:

```suggestion
import (
        "github.com/MakeNowJust/heredoc"
	"github.com/cli/cli/v2/pkg/cmdutil"
	"github.com/spf13/cobra"
)

func NewCmdAutolink(f *cmdutil.Factory) *cobra.Command {
	cmd := &cobra.Command{
		Use:   "autolink <command>",
		Short: "Manage autolink references",
		Long:  heredoc.Docf(`
		Work with GitHub autolink references.
		
		Information about autolinks is only available to repository administrators and can be found at
		https://github.com/{owner}/{repo}/settings/key_links.
		Use %[1]sgh repo autolink list --web%[1]s to open this page for the current repository.
		
		For more information about GitHub autolinks, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/configuring-autolinks-to-reference-external-resources
	`, "`"),
```

Reply:
Resolved via https://github.com/hoffm/github-cli/pull/1, which is now merged in.